### PR TITLE
feat(work-1303): add data-testid to SbModal close button

### PR DESCRIPTION
**Pull request by vcpablo**
## Pull request type

Jira Link: [WORK-1303](https://storyblok.atlassian.net/browse/WORK-1303)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Other (please describe):

## How to test this PR

There's not much to test, since this is a purely QA-oriented ticket.

## What is the new behavior?

- Add dynamic `data-testid` to the `SbModal` close button falling back to `sb-modal-close-button`

## Other information

[WORK-1303]: https://storyblok.atlassian.net/browse/WORK-1303?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/Modal/SbModal.vue [hunk 1]
```diff
@@ -95,7 +103,13 @@ export default {
   },
 
   computed: {
+    closeButtonDataTestId() {
+      const dataTestId = this.$attrs['data-testid']
+      return dataTestId ? `${dataTestId}-close-button` : 'sb-modal-close-button'
+    },
+
     computedClasses() {
+      console.log('io', this.$attrs)
```
Comment (marcosh-ramos) [inline src/components/Modal/SbModal.vue:112]: can we remove this `console.log`?

**Review by marcosh-ramos — APPROVED** [review]
Great! I left only one comment about a console log

**Review by fernandacanto — CHANGES_REQUESTED** [review]
Hi Pablo, I don't think we need this PR, or suddenly we should apply this rule inside the SbModalCloseButton component, what do you think?

The icon button inside this component already has data-testid on the develop branch.

https://github.com/storyblok/storyblok-design-system/blob/develop/src/components/Modal/components/SbModalCloseButton.vue

**Comment by vcpablo** [comment]
> 

We should be able to set it dynamically from outside of the DS component.
This should also be true for other buttons like "Cancel" and "Publish/Unpublish"

I'll leave it up to you to choose the best approach to do it.

**Review by fernandacanto — APPROVED** [review]

**Pull request merged**